app/controllers/user_token_controller.rb

```diff
@@ -0,0 +1,7 @@
+class UserTokenController < Knock::AuthTokenController
+  skip_before_action :verify_authenticity_token, raise: false
+  def create
```

Comment:
Instead of putting this logic in a controller, it's better in a service file. The reason is that controllers are processing CRUD requests normally based on the routing.

Reply:
I am using a routing based authentication and as this controller inherits from Knock::AuthTokenController to access the auth_token method, I am afraid I have to keep this controller